backend/db/db_objects/user_db_objects/UserPersonalData.js

```diff
@@ -0,0 +1,8 @@
+class UserPersonalData {
+  constructor(first_name, second_name, birthday) {
```

Comment:
You probably shouldn't be mixing snake case and camel case for parameters, a new developer coming into the project would be confused as in which one to use. Remember to always be consistent.